schema/frogbot-schema.json

```diff
@@ -110,6 +110,11 @@
         "description": "Handle vulnerabilities with fix versions only.",
         "title": "Handle vulnerabilities with fix versions only"
       },
+      "npmIgnoreDevDependencies": {
+        "type": "boolean",
+        "description": "Allow the option to ignore dev dependencies while scanning npm projects.",
+        "title": "Ignore Npm dev dependencies while scanning"
```

Comment:
Ignore Npm dev dependencies while scanning
-->
Avoid reporting issues for dev dependencies for npm projects